src/CMakeLists.txt

```diff
@@ -637,12 +620,22 @@ if(${WITH_RADOSGW})
     ${CURL_LIBRARIES}
     ${EXPAT_LIBRARIES}
   )
-endif(${WITH_RADOSGW})
-if(${WITH_RBD})
+endif(WITH_RADOSGW)
+if(WITH_RBD)
   set(DENCODER_EXTRALIBS
       ${DENCODER_EXTRALIBS}
       rbd_replay_types)
-endif(${WITH_RBD})
+  if(LINUX)
+    list(APPEND dencoder_srcs
+        $<TARGET_OBJECTS:krbd_objs>)
+  endif(LINUX)
+endif(WITH_RBD)
+if(WITH_KVS)
```

Comment:
@tchaikov 
Could be that I'm misinterpreting things. But the KeyValue store seems to use the file common/secret.c which includes keyutils.h.
And that we don't have in FreeBSD.
And I have not figured out how to patch this up: either write a replacement, and/or stub it up.

I'll take a note of this and put it at least in my freebsd-todo.txt